app/lib/applets/conversations/view/conversations_view.dart

```diff
@@ -476,69 +477,76 @@ class _ConversationsViewState extends State<ConversationsView> {
-                accountType: sph!.session.accountType,
-                builder: (context, data, accountType, settings, updateSetting, refresh) {
-                  return RefreshIndicator(
-                      key: _refreshKey,
-                      edgeOffset: advancedSearch && !toggleMode ? 256 : 64,
-                      onRefresh: refresh!,
-                      child: CustomScrollView(
-                        controller: scrollController,
-                        physics: AlwaysScrollableScrollPhysics(),
-                        slivers: [
-                          PinnedHeaderSliver(
-                            child: ScrolledDownContainer(
-                                child: toggleMode
-                                    ? toggleModeAppBar()
-                                    : searchWidget()),
-                          ),
-                          SliverVariedExtentList.builder(
-                            itemCount: data.length + 1,
-                            itemExtentBuilder: (index, _) {
-                              if (index > data.length - 1) {
-                                return tileSize * 2.5;
-                              }
-
-                              return tileSize;
-                            },
-                            itemBuilder: (context, index) {
-                              if (index > data.length - 1) {
-                                return Padding(
-                                  padding: const EdgeInsets.only(
-                                      top: 12.0, left: 12.0, right: 12.0),
-                                  child: ListTile(
-                                    title: Text(
-                                      AppLocalizations.of(context)!
-                                          .noFurtherEntries,
-                                      textAlign: TextAlign.center,
-                                      style: Theme.of(context)
-                                          .textTheme
-                                          .titleLarge,
-                                    ),
-                                    subtitle: Text(
-                                      AppLocalizations.of(context)!
-                                          .conversationNote,
-                                      textAlign: TextAlign.center,
-                                    ),
-                                  ),
-                                );
-                              }
-
-                              return ConversationTile(
-                                entry: data[index],
-                                toggleMode: toggleMode,
-                                checked: checkedTiles[data[index].id] ??
-                                    false,
-                              );
-                            },
-                          )
-                        ],
-                      ));
-                }),
+            body: StreamBuilder(
```

Comment:
Is this streambuilder realy neccessary? Cant this information be obtained by the themedata itself? Seems a bit off subscribing to a stream here. If the stream is neccecary, do it without the stream builder outside of the build method with an default value.

Reply:
Changed